doc/language/globals.md

```diff
@@ -10,74 +10,75 @@ To use the module:
-| `$<`      | `$DEFAULT_INPUT`            | Default standard input; `ARGF` or `$stdin`.   |
-| `$>`      | `$DEFAULT_OUTPUT`           | Default standard output; initially `$stdout`. |
-| `$.`      | `$INPUT_LINE_NUMBER`, `$NR` | Input position of most recently read stream.  |
-| `$_`      | `$LAST_READ_LINE`           | String from most recently read stream.        |
+|    Variable      |                 \English                  |       Initially       | Read-Only | Set By                |
+|:----------------:|:-----------------------------------------:|:---------------------:|:---------:|-----------------------|
+| <tt>$stdin</tt>  |                                           |    <tt>STDIN</tt>     |    No.    |                       |
+| <tt>$stdout</tt> |                                           |    <tt>STDOUT</tt>    |    No.    |                       |
+| <tt>$stderr</tt> |                                           |    <tt>STDERR</tt>    |    No.    |                       |
+|   <tt>$<</tt>    |          <tt>$DEFAULT_INPUT</tt>          |     <tt>ARGF</tt>     |   Yes.    |                       |
+|   <tt>$></tt>    |         <tt>$DEFAULT_OUTPUT</tt>          |    <tt>STDOUT</tt>    |    No.    |                       |
+|   <tt>$></tt>    |         <tt>$DEFAULT_OUTPUT</tt>          |   <tt>STDOUT </tt>    |    No.    |                       |
+|   <tt>$.</tt>    | <tt>$INPUT_LINE_NUMBER</tt>, <tt>$NR</tt> | Non-negative integer. |    No.    | Certain read methods. |
+|   <tt>$_</tt>    |         <tt>$LAST_READ_LINE</tt>          |     <tt>nil</tt>      |    No.    | Certain read methods. |
 
 ### Processes
 
-| Variable                  | English               | Contains                                               |
-|---------------------------|-----------------------|--------------------------------------------------------|
-| `$0`                      |                       | Initially, the name of the executing program.          |
-| `$*`                      | `$ARGV`               | Points to the `ARGV` array.                            |
-| `$$`                      | `$PROCESS_ID`, `$PID` | Process ID of the current process.                     |
-| `$?`                      | `$CHILD_STATUS`       | Process::Status of most recently exited child process. |
-| `$LOAD_PATH`, `$:`, `$-I` |                       | Array of paths to be searched.                         |
-| `$LOADED_FEATURES`, `$"`  |                       | Array of paths to loaded files.                        |
+|                    Variable                    |                     \English        |       Initially       | Read-Only |
+|:----------------------------------------------:|:-----------------------------------:|:---------------------:|:---------:|
+|                <tt>$0</tt>                     |                                     |     Program name.     |    No.    |
```

Comment:
`$PROGRAM_NAME` is a built-in alias of `$0`.

Reply:
Added.